src/editor/esphome-editor.ts

```diff
@@ -61,6 +61,8 @@ export class ESPHomeEditor extends LitElement {
         enabled: false,
       },
       tabSize: 2,
+      fontFamily:
+        "ui-monospace,SFMono-Regular,Menlo,Monaco,Consolas,Liberation Mono,Courier New,monospace",
```

Comment:
I think that liberation mono still needs to be wrapped in quotes to be a valid font family value, no ?
```suggestion
        "ui-monospace,SFMono-Regular,Menlo,Monaco,Consolas,\"Liberation Mono\",\"Courier New\",monospace",
```

Reply:
you're right, spaces needs to be escaped